packages/portal/backend/src/server/common/ClasslistAgent.ts

```diff
@@ -49,50 +49,68 @@ export class ClasslistAgent {
         }
     }
 
+    private mapPersonToStudent(p: Person) {
```

Comment:
This function already exists `PersonController::personToTransport(person: Person): StudentTransport`

Reply:
The change has been implemented. We can consider moving the function, as the code is being reused, but I am personally happy with where it is right now.